core/src/test/java/org/hibernate/ogm/backendtck/dialectinvocations/GridDialectOperationInvocationForOneToOneTest.java

```diff
@@ -14,6 +14,8 @@
 
 import org.hibernate.Session;
 import org.hibernate.Transaction;
+import org.hibernate.ogm.backendtck.associations.onetoone.Husband;
```

Comment:
Yes, it bothered me too but I didn't like having these tests in different packages either as I think it's interesting to have a global vision of how it works.